mmv1/templates/terraform/examples/network_security_tls_inspection_policy_custom.tf.erb

```diff
@@ -0,0 +1,136 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_privateca_ca_pool" "default" {
+  provider = google-beta
+  name      = "<%= ctx[:vars]['privateca_ca_pool_name'] %>"
+  location  = "us-central1"
+  tier     = "DEVOPS"
+  publishing_options {
+    publish_ca_cert = false
+    publish_crl = false
+  }
+  issuance_policy {
+    maximum_lifetime = "1209600s"
+    baseline_values {
+      ca_options {
+        is_ca = false
+      }
+      key_usage {
+        base_key_usage {}
+        extended_key_usage {
+          server_auth = true
+        }
+      }
+    }
+  }
+}
+
+
+resource "google_privateca_certificate_authority" "default" {
+  provider = google-beta
+  pool = google_privateca_ca_pool.default.name
+  certificate_authority_id = "<%= ctx[:vars]['privateca_certificate_authority_id'] %>"
+  location = "us-central1"
+  lifetime = "86400s"
+  type = "SELF_SIGNED"
+  deletion_protection = false
+  skip_grace_period = true
+  ignore_active_certificates_on_deletion = true
+  config {
+    subject_config {
+      subject {
+        organization = "Test LLC"
+        common_name = "my-ca"
+      }
+    }
+    x509_config {
+      ca_options {
+        is_ca = true
+      }
+      key_usage {
+        base_key_usage {
+          cert_sign = true
+          crl_sign = true
+        }
+        extended_key_usage {
+          server_auth = false
+        }
+      }
+    }
+  }
+  key_spec {
+    algorithm = "RSA_PKCS1_4096_SHA256"
+  }
+}
+
+resource "google_project_service_identity" "ns_sa" {
+  provider = google-beta
+
+  service = "networksecurity.googleapis.com"
+}
+
+resource "google_privateca_ca_pool_iam_member" "tls_inspection_permission" {
+  provider = google-beta
+
+  ca_pool = google_privateca_ca_pool.default.id
+  role = "roles/privateca.certificateManager"
+  member = "serviceAccount:${google_project_service_identity.ns_sa.email}"
+}
+
+resource "google_certificate_manager_trust_config" "default" {
+  provider    = google-beta
+  name        = "<%= ctx[:vars]['trust_config_name'] %>"
+  description = "sample trust config description"
+  location    = "us-central1"
+
+  trust_stores {
+    trust_anchors {
+      pem_certificate = file("test-fixtures/ca_cert.pem")
+    }
+    intermediate_cas {
+      pem_certificate = file("test-fixtures/ca_cert.pem")
+    }
+  }
+
+  labels = {
+    foo = "bar"
+  }
```

Comment:
unless we're not testing the `google_certificate_manager_trust_config ` resource I'd keep it as minimal as possible. For example, I wouldn't put `labels = {...}`